controller/eks-cluster-config-handler.go

```diff
@@ -1326,6 +1326,12 @@ func createNodeGroup(eksConfig *eksv1.EKSClusterConfig, group eksv1.NodeGroup, e
 		},
 	}
 
+	if gpu := group.Gpu; aws.BoolValue(gpu) {
+		nodeGroupCreateInput.AmiType = aws.String(eks.AMITypesAl2X8664Gpu)
+	} else {
+		nodeGroupCreateInput.AmiType = aws.String(eks.AMITypesAl2X8664)
+	}
```

Comment:
nit: this is probably actually the default since we never set it before but I think it doesn't hurt to do this